src/native/libs/configure.cmake

```diff
@@ -562,7 +574,11 @@ elseif(CLR_CMAKE_TARGET_ANDROID)
     unset(HAVE_ALIGNED_ALLOC) # only exists on newer Android
     set(HAVE_CLOCK_MONOTONIC 1)
     set(HAVE_CLOCK_REALTIME 1)
-elseif(CLR_CMAKE_TARGET_BROWSER OR CLR_CMAKE_TARGET_WASI)
+elseif(CLR_CMAKE_TARGET_WASI)
+    set(HAVE_FORK 0)
+    set(HAVE_GETIFADDRS 0)
+    set(HAVE_GETNAMEINFO 0)
```

Comment:
does cmake misdetect the presence of these functions?

Reply:
The function is there, but calling it is no good 🤣 

![image](https://github.com/user-attachments/assets/0b161c26-e68c-448e-8c20-df5cbe403e14)

Reply:
haha nice. might be worth adding a comment that explains that

Reply:
basically use `unset` like we do for ios/tvos above